domain/account/account.go

```diff
@@ -15,28 +15,38 @@ func (auc *UseCase) CreateAccount(name string, cpf string, secret string, balanc
 	if err != nil {
 		return uuid.UUID{}, err
 	}
-	account := auc.GetAccountCPF(cpf)
+	account, err := auc.GetAccountCPF(cpf)
+	if err != nil {
+		return uuid.UUID{}, err
+	}
 	err = CheckAccountExistence(account)
 	if err != nil {
 		return uuid.UUID{}, err
 	}
-	err = domain.CheckAmount(balance)
+	err = CheckBalance(balance)
 	if err != nil {
 		return uuid.UUID{}, domain.ErrBalanceAbsent
 	}
 	aux := 0
 	var id uuid.UUID
 	for aux == 0 {
 		id, _ = domain.Random()
-		if (auc.SearchAccount(id) != domain.Account{}) {
+		searchAccount, err := auc.SearchAccount(id)
+		if err != nil {
+			return uuid.UUID{}, err
+		}
+		if (searchAccount != domain.Account{}) {
 			aux = 0
 		} else {
 			aux = 1
 		}
 	}
```

Comment:
hey @CMedrado , can't figure out what you're trying to do here? Are u just checking if the generated uuid is unique? If so it's not necessary since its collision probability is extremely low